server.js

```diff
@@ -5,13 +5,46 @@ const axios = require('axios');
 const app = express();
 app.use('/homes/:id', express.static(path.join(__dirname, 'public')));
 
-app.use('/homes/:id', (req, res) => {
+app.get('/homes/:id/schools', (req, res) => {
   axios.get(`http://localhost:3002/homes/${req.params.id}/schools`)
     .then((schools) => {
       res.json(schools.data);
     })
     .catch((err) => {
-      console.log('Error in controller GET');
+      console.log('Error in schools GET');
+      console.error(err);
+    });
+});
+
+app.get('/data/homes/:id', (req, res) => {
+  axios.get(`http://localhost:3001/data/homes/${req.params.id}`)
+    .then((similarHomes) => {
+      res.send(similarHomes.data);
+    })
+    .catch((err) => {
+      console.log('Error in similiar homes GET');
+      console.error(err);
+    });
+});
+
+app.get('/homes/:id/schedule', (req, res) => {
+  axios.get(`http://54.193.202.190:3004/homes/${req.params.id}/schedule`)
+    .then((schedule) => {
+      res.send(schedule.data);
+    })
+    .catch((err) => {
+      console.log('Error in similiar homes GET');
+      console.error(err);
+    });
+});
+
+app.post('/homes/:id/schedule', (req, res) => {
+  axios.post(`http://54.193.202.190:3004/homes/${req.params.id}/schedule`)
+    .then((newBooking) => {
+      res.send(newBooking.data);
+    })
+    .catch((err) => {
+      console.log('Error in similiar homes GET');
```

Comment:
I can't speak to exactly how this post request is formatted but I don't believe any data is actually being passed. I think that req.body needs to be sent via axios. I typically use a different format for my axios calls so I'm not positive but try axios.post(<URL>, req.body)

Reply:
typo: axios.post(URL, req.body)